pbs/pbsrequest.go

```diff
@@ -67,23 +67,22 @@ func (bidder *PBSBidder) LookupBidID(Code string) string {
 }
 
 type PBSRequest struct {
-	AccountID     string            `json:"account_id"`
-	Tid           string            `json:"tid"`
-	TimeoutMillis uint64            `json:"timeout_millis"`
-	AdUnits       []AdUnit          `json:"ad_units"`
-	UserIDs       map[string]string `json:"user_ids"`
-	IsDebug       bool              `json:"is_debug"`
+	AccountID     string          `json:"account_id"`
+	Tid           string          `json:"tid"`
+	CacheMarkup   int8            `json:"cache_markup"`
```

Comment:
PBS would store markup in the cache and return the client a unique key to retrieve it. The client then hits the cache with that key to get the markup.